build-tests/api-documenter-test/src/index.ts

```diff
@@ -52,6 +52,28 @@ export function exampleFunction(x: ExampleTypeAlias, y: number): IDocInterface1
   return undefined as unknown as IDocInterface1;
 }
 
+/**
+ * @public
+ */
+export type MyType = {
+  /**
+   * A name example
+   */
+  name: string;
+};
+
+/**
+ * @public
+ */
+export type MyUnionType =
```

Comment:
This type alias will not have its members documented, correct? If that's the case, can we say so in its doc comment to make it clear to readers what this type alias is testing?